### DISCUSSION TITLE
Layout being weird?

### DISCUSSION THREAD

**Discussion by blyxyas** [community/community#54608] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I don't know if this is a bug, or just bad design. But it seems like the front page layout is weirdly uncentered and with a lot of dead space on the right hand.

![image](https://user-images.githubusercontent.com/73757586/236608945-c785cf77-1704-47ce-96f3-cf3338276c6b.png)

This just changed a few days back. If it's A/B testing, it's ugly.

**Comment by frankli0324** [comment]
for reference: 
![image](https://user-images.githubusercontent.com/20221896/236632438-160cda94-ee24-440f-add5-f4636b86a778.png)
